src/Shopsys/ShopBundle/Resources/views/Front/Layout/layoutWithPanel.html.twig

```diff
@@ -13,7 +13,9 @@
         </div>
 
         <div class="web__main__panel">
-            {{ render(controller('ShopsysShopBundle:Front/Category:panel', { request: app.request } )) }}
+            {% cache 'category_panel'~app.request.pathinfo 5*60 %}
```

Comment:
So this is cached individually for every single page? That seems a bit wasteful to me.

Reply:
Yes it is, but in different categories the panel looks different. This is a quick solution